Userland/Libraries/LibTest/TestMain.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021, Andrew Kaster <akaster@serenityos.org>
```

Comment:
One thing I'm unsure about is copyright headers for these files in LibTest where I moved TestSuite.h into. Seems I was inconsistent... TestSuite.h only had andreas on it. Should all these files have Andreas 2018-2020, or wha?